File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/SaleReceipt.java
package io.github.thunkier.thunkmetrc.wrapper.models;import java.util.List;
import com.fasterxml.jackson.annotation.JsonProperty;public class SaleReceipt {
    @JsonProperty("ArchivedDate")
    public String archivedDate;
    @JsonProperty("CaregiverLicenseNumber")
    public String caregiverLicenseNumber;
    @JsonProperty("ExternalReceiptNumber")
    public String externalReceiptNumber;
    @JsonProperty("Id")
    public Integer id;
    @JsonProperty("IdentificationMethod")
    public String identificationMethod;
    @JsonProperty("IsFinal")
    public Boolean isFinal;
    @JsonProperty("LastModified")
    public String lastModified;
    @JsonProperty("PatientLicenseNumber")
    public String patientLicenseNumber;
    @JsonProperty("PatientRegistrationLocationId")
    public Integer patientRegistrationLocationId;
    @JsonProperty("ReceiptNumber")
    public String receiptNumber;
    @JsonProperty("RecordedByUserName")
    public String recordedByUserName;
    @JsonProperty("RecordedDateTime")
    public String recordedDateTime;
    @JsonProperty("SalesCustomerType")
    public String salesCustomerType;
    @JsonProperty("SalesDateTime")
    public String salesDateTime;
    @JsonProperty("TotalPackages")
    public Integer totalPackages;
    @JsonProperty("TotalPrice")
    public Integer totalPrice;
    @JsonProperty("Transactions")
    public List<Object> transactions;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/SaleReceipt.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class SaleReceipt(
    @JsonProperty("ArchivedDate")
    val archivedDate: String? = null,
    @JsonProperty("CaregiverLicenseNumber")
    val caregiverLicenseNumber: String? = null,
    @JsonProperty("ExternalReceiptNumber")
    val externalReceiptNumber: String? = null,
    @JsonProperty("Id")
    val id: Int? = null,
    @JsonProperty("IdentificationMethod")
    val identificationMethod: String? = null,
    @JsonProperty("IsFinal")
    val isFinal: Boolean? = null,
    @JsonProperty("LastModified")
    val lastModified: String? = null,
    @JsonProperty("PatientLicenseNumber")
    val patientLicenseNumber: String? = null,
    @JsonProperty("PatientRegistrationLocationId")
    val patientRegistrationLocationId: Int? = null,
    @JsonProperty("ReceiptNumber")
    val receiptNumber: String? = null,
    @JsonProperty("RecordedByUserName")
    val recordedByUserName: String? = null,
    @JsonProperty("RecordedDateTime")
    val recordedDateTime: String? = null,
    @JsonProperty("SalesCustomerType")
    val salesCustomerType: String? = null,
    @JsonProperty("SalesDateTime")
    val salesDateTime: String? = null,
    @JsonProperty("TotalPackages")
    val totalPackages: Int? = null,
    @JsonProperty("TotalPrice")
    val totalPrice: Int? = null,
    @JsonProperty("Transactions")
    val transactions: List<Any>? = null
)
